miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/signatureverifier/MiniAppZipUtil.kt

```diff
@@ -0,0 +1,37 @@
+package com.rakuten.tech.mobile.miniapp.signatureverifier
+
+import java.io.File
+import java.io.FileOutputStream
+import java.io.InputStream
+
+private const val SUB_DIR_MINIAPP_ZIP = "miniapp/temp_zip/"
+
+internal class MiniAppZipUtil(val basePath: File) {
```

Comment:
this name is confusing I think, because you are not creating a zip archive file, you are just storing the downloaded file input stream in a temp directory. This method class could be more generic and be used by any class wanting to store an inputStream as a temp file

```suggestion
internal class MiniAppFileStreamUtil(val basePath: File) {
```

Reply:
Yeah, I'll update this part.